### ISSUE TITLE
Make "'fail' size encountered."  error more user friendly 

### ISSUE THREAD

**Issue by Oded-B** [chaaz/versio#9]
The current error message isn't very intuitive, especially for users not familiar with Versio.
We have a Terraform module monorepo that get contribution from multiple engineering teams, most of them are not familiar with Versio at all, if they make some mistake in the [conventional] commit messages, they receive this somewhat opaque error message.

I would suggest something like:
"Error: version increment size could not be calculated"
Or even
"Error: version increment size could not be calculated, ensure Conventional Commits were used"

**Comment by chaaz** [comment]
@Oded-B Thanks, that's a good point! I think we can definitely improve that error message. I'll get something more useful out shortly.

**Comment by chaaz** [comment]
@Oded-B I just released 0.6.2 of Versio, which changes the error message to something more readable. When run from the `release` command, you should see something like:
```
Error: Couldn't parse conventional commit(s): 798bd91,a57c27c
```
The error will include up the 5 most recent commits that couldn't be parsed. Let me know if this is workable for you.

**Comment by chaaz** [comment]
:tada: Awesome, I'll close this issue. Please feel free to create another if you have any more issues.
